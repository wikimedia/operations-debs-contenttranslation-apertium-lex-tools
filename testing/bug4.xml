<rules>
  <rule><match lemma="недела" tags="n.f.*"><select lemma="Sunday" tags="n.*"/></match></rule>
</rules>
