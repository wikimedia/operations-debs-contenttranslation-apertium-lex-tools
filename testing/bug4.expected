^во<pr>/in<pr>/at<pr>/on<pr>$ ^недела<n><f><sg><nom><def><@P←>/Sunday<n><sg><nom><def><@P←>$ ^free<prn><pers><p3><mfn><pl><nom><@SUBJ→>/prpers<prn><p3><mfn><pl><nom><@SUBJ→>$ ^clitic<prn><ref><pers><mfn><sp><acc><@X>/<mfn><sp><acc><@X>$ ^состане<vblex><perf><tv><aor><p3><pl>/meet<vblex><aor><p3><pl>$ ^со<pr>/by<pr>/of<pr>/in<pr>/for<pr>/with<pr>/by means of<pr>$ ^естонски<adj><mfn><pl><nom><def>/Estonian<adj><mfn><pl><nom><def>$ ^мировник<n><m><pl><nom><ind>/peacekeeper<n><pl><nom><ind>$ ^во<pr>/in<pr>/at<pr>/on<pr>$ ^Косово<np><top><nt><sg><nom>/Kosovo<np><top><nt><sg><nom>$ ^.<sent>/.<sent>$
